cmd/kops-controller/pkg/server/server.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"hash/fnv"
+	"io/ioutil"
+	"net/http"
+	"runtime/debug"
+	"time"
+
+	"github.com/gorilla/mux"
```

Comment:
If we're on the topic of unneeded complexity ... ;-)

Reply:
Cribbed from node-authorizer. I can take it out.

Reply:
Removed